audiophiler/util.py

```diff
@@ -18,3 +19,7 @@ def wrapped_function(*args, **kwargs):
         kwargs["auth_dict"] = auth_dict
         return func(*args, **kwargs)
     return wrapped_function
+
+def get_tour_lock_status():
+    lock = Tour.query.first()
+    return lock.tour_lock
```

Comment:
not sure if this should be in the utils file

Reply:
Why shouldn't it be? It is there in gallery.

Reply:
alright nvmd then